bg-geocode/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
```

Comment:
@rizzkhan1 also [includes this file in his PR](https://github.com/DFE-Digital/manage-courses-backend/pull/1032/files#diff-ce4e39c0db023b61c0fa12db36c3d60c). My comment there was just that the naming sounds like this is the `manage-courses-backend`. Technically, of course, it is ... but I just wonder if it wouldn't be confusing one day. Just a minor issue, though.